hs_geographic_feature_resource/models.py

```diff
@@ -1,15 +1,20 @@
+from lxml import etree
+
 from django.db import models
 from django.contrib.contenttypes.fields import GenericRelation
 
 from mezzanine.pages.page_processors import processor_for
 
-from hs_core.models import BaseResource, ResourceManager, resource_processor,\
-                           CoreMetaData, AbstractMetaDataElement
+from dominate.tags import legend, table, tbody, tr, td, th, h4, div
 
-from lxml import etree
+from hs_core.models import BaseResource, ResourceManager, resource_processor, \
+    CoreMetaData, AbstractMetaDataElement
 
+from hs_core.hydroshare.utils import add_metadata_element_to_xml
 
-# Create your models here.
+
+# TODO: Pabitra - this model is not being used anymore. however, deleting this class
+# causes the migration to hang during the build process
```

Comment:
@hyi and @mjstealey If I change the hsctl script this line(https://github.com/hydroshare/hydroshare/blob/develop/hsctl#L247) and this line(https://github.com/hydroshare/hydroshare/blob/develop/hsctl#L249) to include the flag (--noinput) then the migrate command runs without hanging. Do you see any issue using this additional flag for the migrate command?